appium/tests/helpers/PublicKeyHelper.ts

```diff
@@ -0,0 +1,40 @@
+import MailFolderScreen from "../screenobjects/mail-folder.screen";
+import NewMessageScreen from "../screenobjects/new-message.screen";
+import MenuBarScreen from "../screenobjects/menu-bar.screen";
+import SettingsScreen from "../screenobjects/settings.screen";
+import ContactScreen from "../screenobjects/contacts.screen";
+import ContactPublicKeyScreen from "../screenobjects/contact-public-key.screen";
+import PublicKeyDetailsScreen from "../screenobjects/public-key-details.screen";
+
+class PublicKeyHelper {
+ static checkSignatureAndFingerprints = async (userEmail: string, signatureDate: string , fingerprintsValue: string ) => {
```

Comment:
should be called `loadRecipientInComposeThenCheckSignatureAndFingerprints` :shrug: 

Or maybe could be split into two methods. `loadRecipientInCompose` and `checkSignatureAndFingerprints`

Reply:
renamed to loadRecipientInComposeThenCheckSignatureAndFingerprints